src/components/soknadsStatus/SoknadsStatus.tsx

```diff
@@ -51,7 +51,7 @@ const SoknadsStatus: React.FC<Props> = ({status, sak, leserData}) => {
             </div>
 
             {status === SoknadsStatusEnum.BEHANDLES_IKKE && (
-                <div className="status_detalj_panel_luft_under">
```

Comment:
Det ser ikke ut til at vi har noe styling for `status_detalj_panel_luft_under` fra tidligere. Ser at denne klassen også er brukt på linje 62. Kan den fjernes der også?